build.gradle

```diff
@@ -48,9 +48,8 @@ project.dependencies {
     implementation "org.apache.logging.log4j:log4j-iostreams:${log4j2Version}"
     api "com.github.lookfirst:sardine:${project.lookfirstSardineVersion}"
     api "com.googlecode.json-simple:json-simple:${jsonSimpleVersion}"
-    //implementation "javax.xml.bind:jaxb-api:${jaxbVersion}"
+    //implementation "jakarta.xml.bind:jakarta.xml.bind-api:${jaxbVersion}"
```

Comment:
Do we need to leave this commented-out line here?

Reply:
I don't know... you're the one who commented it out 🙂

Reply:
Deleted